includes/Uninstaller.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Uninstaller class.
+ *
+ * @link      https://github.com/googleforcreators/web-stories-wp
+ *
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ */
+
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+use Google\Web_Stories\Infrastructure\PluginUninstallAware;
+
+/**
+ * Uninstaller class.
+ */
+class Uninstaller extends Plugin {
```

Comment:
This class now does two very different things:

1. Instantiate services
2. Delete transients/optins on uninstall

Deleting options should be done in its own service class.

Reply:
Move to it's own class now.